cozy.example.yaml

```diff
@@ -295,6 +295,10 @@ csp_allowlist:
   # style:  https://allowed.domain.com/
   # font:   https://allowed.domain.com/
 
+  contexts:
```

Comment:
Does the context CSP replace the default CSP or are they cumulative? IMHO this should be documented at least with a comment here.

Reply:
They are cumulative. And yes, I should add a comment here.